src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -146,7 +148,7 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             logger.WriteLine(MigrationStrings.Process_DeletingSonarLintFolder);
             await fileSystem.DeleteFolderAsync(legacySettings.LegacySonarLintFolderPath);
 
-            progress?.Report(new MigrationProgress(0, 1, "Finished", false));
+            progress?.Report(new MigrationProgress(0, 1, "Finished!", false));
```

Comment:
`Migration finished successfully` ? (with or without the !)

FYI the other possible final messages are:
* `Migration cancelled` and 
* `Error during migration. See the Output Window for more information`